src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -58,6 +58,7 @@
   private static final String GPT_DIV_SELECTOR = "[data-gpt-creative-size]";
   private static final String ARTICLE_COMMENTS_CSS_SELECTOR = "#WikiaArticleFooter";
   private static final String MIDDLE_PREFOOTER_CSS_SELECTOR = "#PREFOOTER_MIDDLE_BOXAD";
+  private static final String AD_SLOT_SELECTOR = "div[id*='%s']";
```

Comment:
Can we use existing selectors? https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L26